.github/workflows/pr_test_helm_chart.yml

```
@@ -81,7 +81,6 @@ jobs:
       - name: Deploy KOF Components
         run: |
           make dev-operators-deploy
-          make dev-ms-deploy
+          make dev-ms-deploy 
```

All good, just one unneeded diff to revert:
```suggestion
          make dev-ms-deploy
```